roles/aci/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+# This role contains tasks specific to aci
```

Comment:
I had described this in our call earlier but I might have not been clear. We should move the aci role as task inside `contiv_network` role which is used for setting up the networking service. The aci specific tasks can be trigger based deploymnent-environment. If you wish to keep aci specific tasks in a separate file, you can add a tasks file in `contiv_network` role itself and include it in main `contiv_network` role. For instance, checkout how it is done here: https://github.com/contiv/ansible/blob/master/roles/ceph-common/tasks/main.yml